module/LearnZF2AjaxImageGallery/src/LearnZF2AjaxImageGallery/Controller/IndexController.php

```diff
@@ -107,10 +110,12 @@ protected function deleteimageAction()
             $data = $request->getPost()->toArray();
 
             if ($request->isXmlHttpRequest()) {
+                // @codeCoverageIgnoreStart
                 if (is_file("public".$data["img"])) {
                     unlink("public".$data["img"]);
                     $status = true;
                 }
+                // @codeCoverageIgnoreEnd
```

Comment:
or, add dedicated tests IndexController unit tests that extends PHPUnit_Framework_TestCase: https://github.com/ZF-Commons/ZfcUser/blob/1.x/tests/ZfcUserTest/Controller/UserControllerTest.php so getRequest() can be mocked